### PR TITLE
[feature] Auto-create Network from Operator in IDFM when missing

### PR DESCRIPTION
**Pull request by woshilapin**

### CODE REVIEW

File: src/documentation/netex_idfm_to_ntfs.md [hunk 1]
```diff
@@ -138,7 +138,16 @@ network_id | Network/@id | This field is prefixed. The technical part of the NeT
 network_name | Network/Name | 
 network_timezone | | Fixed value `Europe/Paris`.
 
-The `network` has a complementary code `source` with the identifier of the original associated `Network`.
+In the case of a line without an associated network, the associated operator is
+used as a network. If there is no associated operator, the line is ignored.
+
+NTFS field | Netex-IDFM element | Mapping rule/Comment
+--- | --- | ---
+network_id | Operator/@id | This field is prefixed.
```
Comment (ArnaudOggy) [inline src/documentation/netex_idfm_to_ntfs.md]: With ```Operator_``` concatenation

Comment (woshilapin) [reply src/documentation/netex_idfm_to_ntfs.md]: Done in 8fbb32d94beaddb42abad4c24938e430ffdf33c6.

**Comment by woshilapin** [comment]
I also bumped the version to `0.16.1` in 50f5db08d411b39617b6ba196c37219a5f8fc9ac.

File: src/netex_idf/lines.rs [hunk 2]
```diff
@@ -140,23 +140,48 @@ fn load_netex_lines(
...
+                        })
+                    })
+                    .map(|network| network.id.clone())
+                    .or_else::<failure::Error, _>(|error| {
+                        warn!(
+                            "{}. Creating a Network from Operator '{}'.",
+                            error, company_id
+                        );
+
+                        let network_id = format!("Operator_{}", company.id);
```
Comment (ArnaudOggy) [inline src/netex_idf/lines.rs]: Fails and crashes if the Network (from Operator) has already been created (by a previous Line with the same Operator).
Check this existence before creating it and push it.
```
WARN Failed to find a child 'RepresentedByGroupRef' in element 'Line'. Creating a Network from Operator '100'.
WARN Failed to find a child 'RepresentedByGroupRef' in element 'Line'. Creating a Network from Operator '100'.
identifier Operator_100 already exists
```

Comment (woshilapin) [reply src/netex_idf/lines.rs]: Done in 0b5eb84de652488ad5b9aab6ba674373dbfb3f31. I also added a unit test for that use case.

**Pull request merged**